features/user_localization.feature

```diff
@@ -6,34 +6,34 @@ Feature: User Operations Localization
   Scenario: Creating a user with duplicate email and Ukrainian language
     Given user with email "test@mail.com" exists
     And with language "uk"
-    And creating user with email "test@mail.com", initials "name surname", password "passWORD1"
+    And creating user with email "test@mail.com", initials "namesurname", password "passWORD1"
```

Comment:
Please revert this file and fix the root cause of the bug with initials validator instead of changing the behat tests